course/migrations/0009_questioncategory_link.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.0.7 on 2020-11-18 07:56
```

Comment:
You can't have more than 1 migration per app per pull request. It happened because you are were testing on your local. You have to delete the migration and make a new one.

Follow these steps to make sure you don't mess up your local database otherwise you have to delete your local database.

1. fake roll back all three migrations (use --fake option)
2. delete the migration files
3. make a new migration (manage.py makemigrations)
4. fake migrate the new migration